File: explorationCode/testbench.c
#include <stdio.h> 
#include <sys/time.h>

int fib(int n) { 
    int a = 0, b = 1, temp, i; 
    if (n == 0) 
        return a; 
    for (i = 2; i <= n; i++) { 
        temp = a + b; 
        a = b; 
        b = temp; 
    } 
    return b;
}

File: explorationCode/testbench.py
import time
import sys

for i in range(10):
    startTime = round(time.time() * 1000)
    for j in range(sys.maxsize-1):
        continue
    stopTime = round(time.time() * 1000)
    runTimes[i] = starTime - stopTime
    print("loop {} done\n".format(i))
